pages/recording/[[...id]].tsx

```diff
@@ -185,6 +184,7 @@ function RecordingPage({
     return (
       <>
         {head}
+        <ObjectPreviewSuspenseCacheAdapter />
```

Comment:
Feels goofy to have to hoist this component all the way to the top of the page. We could probably get away with moving it _a little lower_ but I don't want to miss values. Alternately we could move it outside of React entirely and into our module initialization path.